data/json/recipes/recipe_food.json

```diff
@@ -905,7 +906,7 @@
     "book_learn": [ [ "family_cookbook", 2 ], [ "scots_cookbook", 2 ] ],
     "batch_time_factors": [ 83, 3 ],
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 1 } ],
-    "tools": [ [ [ "surface_heat", 7, "LIST" ] ] ],
+    "tools": [ [ [ "surface_heat", 7, "LIST" ] ], [ ["funnel", -1] ]  ],
```

Comment:
```suggestion
    "tools": [ [ [ "surface_heat", 7, "LIST" ] ], [ [ "funnel", -1 ] ] ],
```

Reply:
Sorry again here, I'll proceed to make the changes as soon as possible.